docs/index.md

```diff
@@ -535,32 +534,40 @@ class LoginResponder
 
 This simple responder has separate methods for each possible state that take the exact parameters necessary to generate the response. The response content will always be in JSON format.
 
-This loose structure around actions, input, and responders allows each project to define exactly the right response for the application. There are no hard rules about how the actions is defined, so long as it takes a request and produces a response.
+This loose structure around actions, input, and responders allows each project to define exactly the right response for the application. There are no hard rules about how the actions are defined, so long as each take a request and produce a response.
```

Comment:
I think the original verb conjugation was correct, as they're in the context of "each" rather than "actions."